### PR TITLE
UIDATIMP-699: Cover ActionMenu component with tests

### PR DESCRIPTION
**Pull request by IhorKoba**
## Purpose

To Cover `<ActionMenu>` and related components with tests

## Approach

- Add `ActionMenu.test.js` for ActionMenu component
- Add `menuTemplate.test.js ` for menuTemplate function
- Add `Default.test.js `, `LinkTo.test.js` and `GroupAction.test.js` for each type of template
- Remove unusable `LastMenu.js ` template

## Refs

https://issues.folio.org/browse/UIDATIMP-699

## Screenshots

image

### CODE REVIEW

**Review by IvanKryzhanovskyi — APPROVED** [review]

File: src/components/ActionMenu/ActionMenu.test.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+  return render(component);
+};
+
+describe('Action menu component', () => {
+  it('should be rendered', () => {
+    expect(renderActionMenu()).toBeDefined();
+  });
+
+  it('has 2 items', () => {
```
Comment (mariia-aloshyna) [inline src/components/ActionMenu/ActionMenu.test.js:34]:
```suggestion
  it('with defined menu items', () => {
```

File: src/components/ActionMenu/ItemTemplates/LinkTo.test.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  describe('when button is clicked', () => {
+    it('then `onClick` function should be called', () => {
+      const { getByText } = renderLinkToItemTemplate();
+
+      fireEvent.click(getByText('Run'));
+
+      expect(menu.onToggle.mock.calls.length).toEqual(1);
+    });
+
+    it('then url should be changed', () => {
```
Comment (mariia-aloshyna) [inline src/components/ActionMenu/ItemTemplates/LinkTo.test.js:48]:
```suggestion
    it('and url should be changed', () => {
```

File: src/components/ActionMenu/ActionMenu.test.js (hunk 1 above)
Comment (IhorKoba) [reply src/components/ActionMenu/ActionMenu.test.js]: done

File: src/components/ActionMenu/ItemTemplates/LinkTo.test.js (hunk 2 above)
Comment (IhorKoba) [reply src/components/ActionMenu/ItemTemplates/LinkTo.test.js]: done

**Review by mariia-aloshyna — APPROVED** [review]

**Pull request merged**